expo/expo
# [dev-menu] Fix crash from inspector request failures

# Why

Fix crash from inspector request failures

# How

Handle exceptions from inspector requests.

# Test Plan

1. Load a local app from dev-launcher
2. Stop local metro server
3. Open dev-menu and see if crash happens

# Checklist
